firebase-config/src/main/java/com/google/firebase/remoteconfig/FirebaseRemoteConfig.java

```diff
@@ -66,10 +66,10 @@
    * <p>{@link FirebaseRemoteConfig} uses the default {@link FirebaseApp}, so if no {@link
    * FirebaseApp} has been initialized yet, this method throws an {@link IllegalStateException}.
    *
-   * <p>To identify the current app instance, the fetch request creates a Firebase Instance ID
-   * token, which periodically sends data to the Firebase backend. To stop the periodic sync, call
-   * {@link com.google.firebase.iid.FirebaseInstanceId#deleteInstanceId}. To create a new token and
-   * resume the periodic sync, call {@code fetchConfig} again.
+   * <p>To identify the current app instance, the fetch request creates a Firebase Installations ID,
```

Comment:
Strictly speaking, we are trying to use lowercase "Firebase installation ID." 

This fights our tendency to productize every single thing from Notifications to Extensions, and it is singular, as a reference to *this* installation vs. the product offering overall.